src/NuGet.Core/NuGet.Resolver/PackageResolver.cs

```diff
@@ -116,8 +117,14 @@ public IEnumerable<PackageIdentity> Resolve(PackageResolverContext context, Canc
             // Run solver
             var comparer = new ResolverComparer(context.DependencyBehavior, context.PreferredVersions, context.TargetIds);
 
+            var startTimer = DateTime.Now;
```

Comment:
Consider using `System.Diagnostics.Stopwatch` for perf measurements. It's more precise and convenient.